### PR TITLE
Use text cursor for 'Jump to chat'

### PR DESCRIPTION
**Pull request by akalin-keybase**
Also use stylesheets.

### CODE REVIEW

**Comment by akalin-keybase** [comment]
not for release

File: shared/chat/inbox/row/chat-filter-row/index.js [hunk 1]
```diff
@@ -162,22 +159,25 @@ const styleContainer = {
...
+    alignItems: 'center',
+    backgroundColor: globalColors.black_05,
+    borderRadius: 19,
+    flexGrow: 1,
+    height: 24,
+    justifyContent: 'center',
+    marginRight: globalMargins.small,
+  },
+  isElectron: {
+    cursor: 'text',
```
Comment (chrisnojima) [inline shared/chat/inbox/row/chat-filter-row/index.js]: we have desktopStyles.editable for this

Comment (akalin-keybase) [reply shared/chat/inbox/row/chat-filter-row/index.js]: Fixed

**Comment by akalin-keybase** [comment]
PTAL @chrisnojima

File: shared/chat/inbox/row/chat-filter-row/index.js [hunk 2]
```diff
@@ -162,22 +167,25 @@ const styleContainer = {
...
-  justifyContent: 'center',
-  marginRight: globalMargins.small,
-}
-
-const styleFilterContainerMobile = {
-  ...styleFilterContainer,
-  height: 32,
-  marginRight: globalMargins.small,
-}
+const styleFilterContainer = collapseStyles([
```
Comment (chrisnojima) [inline shared/chat/inbox/row/chat-filter-row/index.js]: we should use styleSheetCreate, only merge desktopStyles.editable if isElectron

Comment (akalin-keybase) [reply shared/chat/inbox/row/chat-filter-row/index.js]: Done.

**Comment by akalin-keybase** [comment]
Okay, PTAL @chrisnojima

**Pull request merged**